src/status_im2/contexts/syncing/enter_sync_code/style.cljs

```diff
@@ -0,0 +1,72 @@
+(ns status-im2.contexts.syncing.enter-sync-code.style
+  (:require [quo2.foundations.colors :as colors]
+            [quo2.foundations.typography :as typography]
+            [react-native.platform :as platform]))
+
+(def container-text-input
+  {:flex-direction     :row
+   :justify-content    :space-between
+   :padding-horizontal 20})
+
+(defn text-input-container
+  [invalid?]
+  {:padding-top      1
+   :padding-left     12
+   :padding-right    7
+   :padding-bottom   7
+   :flex             1
+   :flex-direction   :row
+   :background-color colors/neutral-95
```

Comment:
The background color should be `:transparent` as per the design.

<img src="https://github.com/status-im/status-mobile/assets/19339952/27cb2596-8cde-4634-845e-264e7718e8e3" width="250" />

Reply:
ouch! nice spot! Thanks, I will fix this @smohamedjavid  
I only did my verifications after logging in, I missed checking this from the logged out path.